src/writeJson.py

```diff
@@ -633,36 +633,70 @@
-source_id[key]['institution_id'] = 'PCMDI'
-source_id[key]['institution'] = 'Program for Climate Model Diagnosis and Intercomparison, Lawrence Livermore National Laboratory, Livermore, CA 94550, USA'
-source_id[key]['nominal_resolution'] = '1x1 degree'
-source_id[key]['product'] = 'observations'
-source_id[key]['references'] = ''.join(['Taylor, K.E., D. Williamson and F. Zwiers, ',
-                                        '2000: The sea surface temperature and sea ice ',
-                                        'concentration boundary conditions for AMIP II ',
-                                        'simulations. PCMDI Report 60, Program for ',
-                                        'Climate Model Diagnosis and Intercomparison, ',
-                                        'Lawrence Livermore National Laboratory, 25 pp. ',
-                                        'Available online: https://pcmdi.llnl.gov/report/pdf/60.pdf'])
-source_id[key]['region'] = ['global_ocean']
-source_id[key]['release_year'] = '2019'
-source_id[key]['source_description'] = 'Sea surface temperature and sea-ice datasets produced by PCMDI (LLNL) for the AMIP (DECK) experiment of CMIP6'
-source_id[key]['source'] = 'PCMDI-AMIP 1.1.6: Merged SST based on UK MetOffice HadISST and NCEP OI2'
-source_id[key]['source_id'] = key
-source_id[key]['source_type'] = 'satellite_blended'
-source_id[key]['source_variables'] = ['areacello','sftof','siconc','siconcbcs',
-                                      'tos','tosbcs']
-source_id[key]['source_version'] = '1.1.6'
-source_id[key]['target_mip'] = 'CMIP'
-source_id[key]['title'] = 'PCMDI-AMIP 1.1.6 dataset prepared for input4MIPs'
+# Loop over ISMIP6 source_id entries
+sIds = [
+ 'ACCESS1-3-rcp85-1-0',
+ 'CCSM4-rcp26-1-0',
+ 'CCSM4-rcp85-1-0',
+ 'CESM2-ssp585-1-0',
+ 'CNRM-CM6-1-ssp126-1-0',
+ 'CNRM-CM6-1-ssp585-1-0',
+ 'CNRM-ESM2-1-ssp585-1-0',
+ 'CSIRO-MK3-6-0-rcp85-1-0',
+ 'HadGEM2-ES-rcp85-1-0',
+ 'IPSL-CM5A-MR-rcp26-1-0',
+ 'IPSL-CM5A-MR-rcp85-1-0',
+ 'MIROC-ESM-CHEM-rcp26-1-0',
+ 'MIROC-ESM-CHEM-rcp85-1-0',
+ 'MIROC5-rcp26-1-0',
+ 'MIROC5-rcp85-1-0',
+ 'NorESM1-M-rcp26-1-0',
+ 'NorESM1-M-rcp85-1-0',
+ 'UKESM1-0-LL-ssp585-1-0'
+ ]
+
+for count,sId in enumerate(sIds):
+    key = sId
+    tmp = key.replace('-1-0','')
+    rcp = tmp.split('-')[-1]
+    mod = tmp.replace(''.join(['-',rcp]),'')
+    print(count,key,mod,rcp)
+    #sys.exit()
+    source_id[key] = {}
+    source_id[key]['comment'] = ' '.join(['Prepared using CMIP5 model', mod ,
+                                          'as input. A combination of historical',
+                                          'and', rcp ,'datasets were used to',
+                                          'create this ISMIP6 forcing dataset'])
+    source_id[key]['contact'] = 'ISMIP6 Steering Team (ismip6@gmail.com)'
+    source_id[key]['dataset_category'] = 'atmosphericState'
+    source_id[key]['grid'] = '1x1 degree longitude x latitude'
```

Comment:
This will change depending on if it is Greenland or Antarctica. This will be specified in the global attributes from the user table when I write the file with CMOR.

Reply:
@geresie, no problem, all grid references removed from the registration

Reply:
Perfect, these entries will be removed from the source_id registration